BuildCustomFunctions.ps1

```diff
@@ -396,6 +359,14 @@ $($IndentChar)$($IndentChar)$($IndentChar)$($IndentChar)Return `$ResponseTask
 $($IndentChar)$($IndentChar)$($IndentChar)}
 $($IndentChar)$($IndentChar))"
                 # Build "Process" block
+                # temp fix for Set-JcSdkUserAssociation TODO: remove when DE-2630 is done
```

Comment:
For Set-JCsdkUserAccoc. if we remove an association , we need to pass in null attributes until DE-2630 is resolved